tailwind.config.js

```diff
@@ -0,0 +1,50 @@
+const variableTokens = require('./lib/tokens/json/css-variables-nested.json');
+const staticTokens = require('./lib/tokens/json/variables-nested.json');
+
+module.exports = {
```

Comment:
copied from LP (https://github.com/FB-PLP/traject/blob/master/tailwind.config.js), minus the `extend` block (not sure those are valid for EDS anymore). after our next publish we probably want update traject's config to read from this file again